py3status/modules/imap.py

```diff
@@ -27,17 +29,24 @@ def check_mail(self, i3s_output_list, i3s_config):
         mail_count = self._get_mail_count()
 
         response = {
-            'cached_until': time() + self.cache_timeout,
-            'full_text': '{}: {}'.format(self.name, mail_count)
+            'cached_until': time() + self.cache_timeout
         }
 
-        new_mail_color = i3s_config['color_good']
-        check_failed_color = i3s_config['color_bad']
+        if not self.new_mail_color:
+            self.new_mail_color = i3s_config['color_bad']
```

Comment:
Feel free to change it -- I used it this way because I like to be highlighted when there are new mails, and I only actively notice things when they indicate a problem :D